experiments/integrated/fluxnet/data_tools.jl

```diff
@@ -63,28 +63,7 @@ struct DataColumn
     status::DataStatus
 end
 
-function DataColumn(file_matrix::Matrix, column_name::String, units::String)
-    """Returns a DataColumn struct containing the data from the column
-    specified by the string column_name, along with the units of the data and 
-    a status flag which indicates the data is complete, missing, or
-    incomplete."""
-    col_dat, status = check_column(file_matrix, column_name)
-    # Patch missing values with mean
-    if status == incomplete
-        num_missing = count(x -> x == -9999, col_dat)
-        percent_missing = 100.0 * num_missing / length(col_dat)
-        if percent_missing > 90.0
-            return DataColumn(col_dat, units, absent)
-        end
-        @info "Warning: Data for $column_name $(round(percent_missing,
-        sigdigits=3))% missing. Filling with mean value"
-        replace_missing_with_mean_by_value!(col_dat)
-        status = complete
-    end
-    return DataColumn(col_dat, units, status)
-end
-
-function VerifiedColumn(file_matrix::Matrix, column_name::String, units::String)
+function filter_column(file_matrix::Matrix, column_name::String, units::String)
```

Comment:
Are these changes related to the boundary vars change? It might be nice to have them in a separate PR if not, but I don't think it's a huge deal either way

Reply:
good point!  @Espeer5 I will make these changes in a separate PR this week